### DISCUSSION TITLE
Custom plugins

### DISCUSSION THREAD

**Discussion by goldzulu** [FlowiseAI/Flowise#100] [Q&A]
Hi, I am writing my own own plug-in at the moment coz can't find one that suits my needs at the moment. It's not specifically a langchain plugin but more an Autogpt one. 

I think I am making some progressive hack but please can I get some directions on how do this. Will write a blog and YouTube video on this if successful.

Many thanks!

**Comment by HenryHengZJ** [comment]
Hey we would definitely love to have a successful use case of adding custom plugin! Feel free to hop into Discord and we can assist you there 😉
